charts/castai-db-optimizer/ci/test-values.yaml

```diff
@@ -4,3 +4,6 @@ apiKey: "456"
 resources:
   proxy:
     ephemeralStorage: 5Mi
+
+proxy:
+  readinessProbeEnabled: false
```

Comment:
Why we don't care about this in CI?

Reply:
Because the CI test step is waiting for the cache to be ready and since it's not using real data it's failing with `Readiness probe failed: Initial cache configuration wasn't pulled yet` - https://github.com/castai/helm-charts/actions/runs/11271686870/job/31345023475